tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -135,15 +123,7 @@ def tap_collection(self, collection_name):
 
     @property
     def app_elements(self):
-        return self.marionette.execute_script("""
-        var gridItems = window.wrappedJSObject.app.grid.getItems();
-        var appElements = [];
-        for(var i=0; i<gridItems.length; i++){
-        // it must have an app to be a
-        if(gridItems[i].app) appElements.push(gridItems[i].element);
-        }
-        return appElements;
-        """)
+        return self.marionette.find_elements(*self._icons_locator)
```

Comment:
oh this part, I just pasted it based on Johan's suggestion to avoid conflict with his PR.  I actually don't use this method.